upi/openstack/inventory.yaml

```diff
@@ -4,8 +4,15 @@ all:
       ansible_connection: local
       ansible_python_interpreter: "{{ansible_playbook_python}}"
 
+      # User-provided values
       os_subnet_range: '10.0.0.0/16'
       os_cluster_name: 'openshift'
+      os_flavor_worker: 'm1.large'
+      os_image_rhcos: 'rhcos'
+
+      # Number of provisioned Compute nodes.
+      # 3 is the minimum number for a fully-functional cluster.
+      os_compute_nodes_numer: 3
```

Comment:
Typo: `numer` -> `number` (needs to be fixed in the playbooks too).

Reply:
Amazing that I had the same typo both sides.